testui/elements/testui_element.py

```diff
@@ -524,12 +520,12 @@ def screenshot(self, image_name="cropped_image.png"):
         return self
 
     def find_image_match(
-        self,
-        image_name,
-        threshold=0.9,
-        image_match="",
-        max_scale=2.0,
-        min_scale=0.3,
+            self,
```

Comment:
These formatting changes in this file are not necessary, they were correct before.

Reply:
yes, fixed that